client/css/style.css

```diff
@@ -818,6 +819,11 @@ button,
 	color: #2ecc40;
 }
 
+#chat .ctcp .from:before {
+	font-family: FontAwesome;
+	content: "\f0f6";
+}
+
```

Comment:
Fair enough, and therefore even better now then (code separation + allegedly better handling on the client)!
Thanks for specifying :-)